docker-compose.yml

```diff
@@ -23,4 +23,6 @@ services:
       - "./frontend:/app:rw"
       - "./scripts:/app/scripts"
     environment:
-      - BACKEND_PROXY=http://backend:3001
+      - JEST_JUNIT_OUTPUT_DIR=reports
+      - JEST_JUNIT_OUTPUT_NAME=unit.xml
```

Comment:
These probably got added back in during a merge but `JEST_JUNIT*` vars can be removed from this file.

Reply:
Removed